include/seqan3/core/concept/core_language.hpp

```diff
@@ -143,6 +143,19 @@ template <typename t>
 concept floating_point_concept = arithmetic_concept<t> && std::is_floating_point_v<t>;
 //!\endcond
 
+/*!\interface   seqan3::char_concept <>
+ * \extends     std::Integral
+ * \brief       This concept encompasses exactly the types `char`, `signed char`, `unsigned char`, `wchar_t`,
+ *              `char16_t` and `char32_t`.
+ */
+//!\cond
+
+template <typename t>
+concept char_concept = std::Integral<t> &&
```

Comment:
isn't requiring `std::Integral<t>` obsolete if you test for the exact type afterward?

Reply:
But I want char_concept to more refined than Integral so you can use these for specialisation.